synfig-studio/src/gui/dialogs/dialog_guide.cpp

```diff
@@ -0,0 +1,261 @@
+**	along with Synfig.  If not, see <https://www.gnu.org/licenses/>.
+**	\endlegal
+*/
+/* ========================================================================= */
+
+/* === H E A D E R S ======================================================= */
+
+#ifdef USING_PCH
+#	include "pch.h"
+#else
+#ifdef HAVE_CONFIG_H
+#	include <config.h>
+#endif
+
+#include <gui/dialogs/dialog_guide.h>
+
+#include <gui/localization.h>
+
+#include <gtkmm/box.h>
+#include <gtkmm/frame.h>
+#include <gtkmm/grid.h>
+
+#include <gui/workarea.h>
+#include <gui/duckmatic.h>
+#include <gui/app.h>
+
+#endif
+
+using namespace synfig;
+using namespace studio;
+
+/* === M A C R O S ========================================================= */
+
+/* === G L O B A L S ======================================================= */
+/* === P R O C E D U R E S ================================================= */
+
+/* === M E T H O D S ======================================================= */
+
+Dialog_Guide::Dialog_Guide(Gtk::Window& parent, etl::handle<synfig::Canvas> canvas, WorkArea *work_area):
+	Dialog(_("Guide Editor"),parent),
+	canvas(canvas),
+	current_work_area(work_area),
+	angle_adjustment(Gtk::Adjustment::create(0,-2000000000,2000000000,1,1,0)),
+/*	center_x_widget_adjust (Gtk::Adjustment::create(0,-2000000000,2000000000,1,1,0)),
+	center_y_widget_adjust (Gtk::Adjustment::create(0,-2000000000,2000000000,1,1,0)),
+	point_x_widget_adjust (Gtk::Adjustment::create(0,-2000000000,2000000000,1,1,0)),
+	point_y_widget_adjust (Gtk::Adjustment::create(0,-2000000000,2000000000,1,1,0)), */
+	degrees(true)
+{
+	this->set_resizable(false);
+	assert(canvas);
+
+	//Box start
+	Gtk::Box *guide_box = Gtk::manage(new Gtk::Box(Gtk::ORIENTATION_VERTICAL));
+
+	angle_widget=manage(new class Gtk::SpinButton(angle_adjustment,15,2));
+	angle_widget->show();
+
+/*	center_x_widget=manage(new class Gtk::SpinButton(center_x_widget_adjust,15,2));
+	center_x_widget->show();
+	center_x_widget->set_hexpand();
+	center_y_widget=manage(new class Gtk::SpinButton(center_y_widget_adjust,15,2));
+	center_y_widget->show();
+	center_y_widget->set_hexpand();
+	point_x_widget=manage(new class Gtk::SpinButton(point_x_widget_adjust,15,2));
+	point_x_widget->show();
+	point_x_widget->set_hexpand();
+	point_y_widget=manage(new class Gtk::SpinButton(point_y_widget_adjust,15,2));
+	point_y_widget->show();
+	point_y_widget->set_hexpand(); */
+
+	Gtk::Frame *angleFrame = manage(new Gtk::Frame(_("Rotate By Setting an Angle")));
+	angleFrame->set_shadow_type(Gtk::SHADOW_NONE);
+	((Gtk::Label *) angleFrame->get_label_widget())->set_markup(_("<b>Rotate By Setting an Angle</b>"));
+	angleFrame->set_margin_bottom(5);
+	angleFrame->set_margin_top(5);
+
+
+	auto guideGrid = manage(new Gtk::Grid());
+	guideGrid->get_style_context()->add_class("dialog-secondary-content");
+	guideGrid->set_row_spacing(6);
+	guideGrid->set_column_spacing(8);
+
+	angle_type_picker.append(_("Degree"));
+	angle_type_picker.append(_("Radian"));
```

Comment:
use `.append("degree", _("Degree"))`.
Otherwise, the comparison in `set_angle_type()` will depend on translation and fail (as it is now)